src/index.js

```diff
@@ -149,6 +151,18 @@ class ReactTooltip extends React.Component {
     this.injectStyles(); // Inject styles for each DOM root having tooltip.
   }
 
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (
+      prevState.show &&
+      this.state.show &&
+      this.content &&
+      this.content !== this.prevContent
```

Comment:
Note to reviewers: I wanted to use a state variable instead of new instance variables so this could be this.state.content !== prevState.content instead, but I got errors and saw recommendations not to call setState() from within render(). If there's a better way to achieve this let me know.